### PR TITLE
Update changelog [ci skip]

### PR DESCRIPTION
**Pull request by pocmo**
A couple of things I haven't added yet.

### CODE REVIEW

File: docs/changelog.md [hunk 1]
```diff
@@ -11,13 +11,22 @@ permalink: /changelog/
...
   * Mozilla App Services (FxA: **0.10.0**, Sync Logins: **0.10.0**, Places: **0.10.0**)
 
+* ⚠️ **This is the first release compiled against Android SDK 28.**
+
 * **lib-crash**
   * The state of the "Send crash report" checkbox is now getting saved and restored once the dialog is shown again. 
+  * The crash reporter can now sends reports if the prompt is closed by pressing the back button.
+
+* **lib-fetch-httpurlconnection**
+  * 🆕 New component: `concept-fetch` implementation using [HttpURLConnection](ad https://developer.android.com/reference/java/net/HttpURLConnection.html).
```
Comment (Amejia481) [inline docs/changelog.md]: Nit: the `ad` character is breaking the `HttpURLConnection` link

Comment (pocmo) [reply docs/changelog.md]: Oops. Advertisment for https://addons.mozilla.org/en-US/firefox/addon/android-sdk-search-fx/ 🙃

Comment (Amejia481) [reply docs/changelog.md]: 💪 This look similar to: 

https://play.google.com/store/apps/details?id=com.jakewharton.sdksearch

https://chrome.google.com/webstore/detail/android-sdk-search/hgcbffeicehlpmgmnhnkjbjoldkfhoin

Nice to have it on 🔥 🦊 !

Comment (pocmo) [reply docs/changelog.md]: Yep, it's a port of Roman's extensions: https://github.com/pocmo/AndroidSDKSearchExtension-Firefox

:)

Comment (Amejia481) [reply docs/changelog.md]: 👍

**Pull request merged**